cdap-app-templates/cdap-program-report/src/main/java/co/cask/cdap/report/proto/summary/ReportSummary.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *  
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *  
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.report.proto.summary;
+
+import com.google.common.base.Objects;
+
+import java.util.Set;
+
+/**
+ * Represents the summary of a program operation status report in an HTTP response.
+ */
+public class ReportSummary {
+  private final Set<NamespaceAggregate> namespaces;
+  private final long start;
+  private final long end;
```

Comment:
i don't see start and end in the design doc object, are you sure its needed

Reply:
in UI, it displays the time range in summary, let me confirm if they are needed in UI

Reply:
just confirmed it's needed. updating the design

Reply:
why is it needed ?